addons/web/static/src/core/user_service.js

```diff
@@ -75,6 +75,7 @@ export const userService = {
             home_action_id,
             show_effect: showEffect,
         } = info;
+        const web_base_url = odoo.session_info['web.base.url'];
```

Comment:
not sure about this

Reply:
Where would you put it ? In another service ? It seemed like most of the session_info stuff was put inside the user service so I did the same but I agree the web_base_url doesn't really belong there. 
Or maybe should I call `odoo.session_info['web.base.url']`straight from the component, bypassing everything ?